v2/tools/generator/internal/config/type_configuration.go

```diff
@@ -295,9 +295,24 @@ func (tc *TypeConfiguration) UnmarshalYAML(value *yaml.Node) error {
 		// - secret2
 		if strings.EqualFold(lastId, azureGeneratedSecretsTag) && c.Kind == yaml.SequenceNode {
 			var azureGeneratedSecrets []string
+			var errs []error
```

Comment:
minor: I'm not totally sold that a collection of errors is actually necessary. For parsing errors, I'd sorta expect the error to come from the beginning of the problem, not all following lines/elements.

Reply:
Fair call, have simplified to return on the first error.